includes/admin/class-wc-rest-stripe-settings-controller.php

```diff
@@ -388,6 +414,29 @@ private function update_is_test_mode_enabled( WP_REST_Request $request ) {
 		$this->gateway->update_option( 'testmode', $is_test_mode_enabled ? 'yes' : 'no' );
 	}
 
+	/**
+	 * Get the merchant payment method configuration in Stripe.
+	 *
+	 * @return array|null
```

Comment:
Looks like this function returns an object instead of an array.

```suggestion
	 * @return object|null
```

Reply:
Thank you @malithsen ! Updated [ebb6e90](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4091/commits/ebb6e90885011cb846191dd4f0a727935676bd9e)